tests/test_snapshot_checkpoint.py

```diff
@@ -2,12 +2,12 @@
 
 from mpi4py import MPI
 
-import adios2
 import basix.ufl
 import dolfinx
 import numpy as np
 import pytest
 
+from adios4dolfinx.adios2_helpers import adios2
```

Comment:
Not sure I like this, could we rather add a similar import fix here?

Reply:
Is this style acceptable within the library, though?

Reply:
Do we use it anywhere else? 
To me it is clearest if we import packages as their origin. Or maybe make a convenience util for it?

def import_adios():
     Try:
        Return ….
     Except: 
        Return ….
? (On my phone , so sorry for pseudocode